project/build.sbt

```diff
@@ -1,4 +1,5 @@
 addSbtPlugin("com.jsuereth" %% "sbt-pgp" % "1.1.0-M1")
+addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "2.0")
```

Comment:
Use sbt-release-early https://github.com/scalacenter/sbt-release-early/ :smile:

Reply:
@jvican I tried but either I did something wrong or I hit some bug in it.This is what I tied to do ([PR](https://github.com/romanowski/hoarder/pull/61/files),  single [commit to master](https://github.com/romanowski/hoarder/commit/398c0198afe6a41f02a4f792a815edd2227afb3a) and [failed build](https://travis-ci.org/romanowski/hoarder/builds/295671990?utm_source=github_status&utm_medium=notification)). Of course all those work is work in progress.